### ISSUE TITLE
Prijímať odpovede testovača cez REST

### ISSUE THREAD

**Issue by maaario** [trojsten/web#1156]
Chce sa toto spraviť alebo by mohol testovať dostávať submity do cloudu stále cez `scp`?

**Comment by mhozza** [comment]
Existuju protokoly, ktore nestracaju spravy. Napr, [rabbitmq](https://www.rabbitmq.com/), ktore je default pre novy celery. Mozme ho vyskusat pouzit priamo, alebo mozme testovac obalit celerom.
Ak to je overkill, tak si testovac moze drzat zoznam messagov, ktore poslal a nedostal na ne od webu odpoved a opakovat ich alebo aspon umoznit ich poslat znovu manualne.

**Comment by maaario** [comment]
No, po tom, co testovac odosle protokol, testovac uz od webu nic nedostane.
Ale opacne by to slo - web si pamata, co odoslal a ci na to dostal odpoved a ak nie, tak by mohol po case od testovaca chciet dostat odpoved znova (asi nie resubmitnut, ale len sa spytat, ci ma vysledok...) pripadne po case vytimeoutit (10 min) a ponuknut resubmit. Ak by od testovaca dodatocne dostal vysledok (po timeoute).

Momentalne je problem aj to, ze protokol ide dvoma kanalmi: testovac scpckuje subor a potom zavola update_submit na webe ... a navyse web polluje ci uz dostal submit ... ked to pojde jednym kanalom zbavime sa potencialnych problemov s nedocitanym protokolom. Takze vlastne scp urcite nechceme, lebo testovac chce oznamit webu vysledok explicitne.
